src/apollo.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
```

Comment:
Could we possibly move this file into the GraphQL folder? I would like to add some stuff as part of the reception story so it would be nice if it was all in one place.